CHANGELOG.md

```diff
@@ -4,15 +4,13 @@ Changes for users of the library currently on `develop`:
 
 _This space intentionally left blank._
 
-## [1.1.0](https://github.com/NYTimes/ios-360-videos/releases/tag/1.1.0)
-
-- Point camera toward correct video center to start ([#49](https://github.com/NYTimes/ios-360-videos/pull/49))
```

Comment:
we probably should keep these in the history, i can't think of a good reason to remove

Reply:
I removed these because it looks like they were made in the original repo and don't really pertain to the codebase as it's presented in our own fork. I did leave in the `Older Releases` section at the bottom of the changelog though, what do you think? Would you still rather keep it in?